Makefile

```diff
@@ -209,7 +209,7 @@ check: ./golangcilint.yaml $(GOLINTER)
 
 swagger/docs.go: 
 	swag -v || go install github.com/swaggo/swag/cmd/swag@$(SWAGGER_VERSION)
-	swag init --parseDependency -o swagger -g pkg/api/routes.go -q
+	swag init --parseDependency -o swagger -g pkg/api/routes.go -g pkg/extensions/extension_mgmt.go -q
```

Comment:
Does this work without adding pkg/extensions/extension_userprefs.go here?

Reply:
I tested and it works without it so I removed it for mgmt too

Reply:
Thanks